x-pack/plugins/security/public/account_management/api_keys_page/api_keys_table.tsx

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+// @ts-ignore
+import { formatDate } from '@elastic/eui/lib/services/format';
+import {
+  EuiBadge,
+  EuiButtonEmpty,
+  EuiHideFor,
+  EuiInMemoryTable,
+  EuiInMemoryTableProps,
+  EuiShowFor,
+  EuiText,
+  EuiButtonIcon,
+} from '@elastic/eui';
+import { ApiKey } from '../../../common/model';
+
+export type ApiKeysTableProps = Omit<EuiInMemoryTableProps<ApiKey>, 'columns'> & {
+  createdItemId?: ApiKey['id'];
+  onInvalidateItem(item: ApiKey): void;
+};
+
+export const ApiKeysTable: FunctionComponent<ApiKeysTableProps> = ({
+  createdItemId,
+  onInvalidateItem,
+  ...props
+}) => {
+  const actions = [
+    {
+      render: (item: ApiKey) => (
+        <>
+          <EuiShowFor sizes={['xs', 's']}>
+            <EuiButtonIcon
+              iconType="minusInCircle"
```

Comment:
nit: we may want either an `aria-label` here or an [EuiTooltip](https://elastic.github.io/eui/#/display/tooltip) so to give users some more context. The tooltip would benefit all users, while the `aria-label` would be a good hint for folks using screen readers. I think either would be fine, but we should use one of them

Reply:
Good idea, I didn't think about that!